Formula/apache-arrow.rb

```diff
@@ -30,9 +30,13 @@ class ApacheArrow < Formula
   depends_on "re2"
   depends_on "snappy"
   depends_on "thrift"
+  depends_on "utf8proc"
   depends_on "zstd"
 
   def install
+    # https://github.com/Homebrew/homebrew-core/issues/76537
+    ENV.runtime_cpu_detection if Hardware::CPU.intel?
```

Comment:
@carlocab all `ENV.runtime_cpu_detection` calls will be gates by an `if Hardware::CPU.intel?`, right? What happens if you call it on ARM, do things break? I almost wonder if it just makes more sense to make it a no-op on ARM so it doesn't need the if statements

Reply:
Not necessarily -- I think it should also be safe to use unconditionally for formulae like `openblas`. I just don't know of a good way to actually inspect the binary objects to check that it probably does the right thing on ARM.